mittab/libs/backup/__init__.py

```diff
@@ -44,6 +44,9 @@ def backup_round(key=None, round_number=None, btime=None):
         if round_number is None:
             round_number = TabSettings.get("cur_round", "no-round-number")
 
+        elif round_number == 0:
```

Comment:
Is this meant to be a part of a different work related to the backup/restore issues?